marlowe-dashboard-client/src/Css.purs

```diff
@@ -47,7 +48,22 @@ bgBlueGradient = [ "bg-gradient-to-r", "from-purple", "to-lightpurple", "text-wh
 
 -- buttons
 button :: Array String
-button = [ "leading-none", "whitespace-nowrap", "px-6", "py-4", "font-black", "rounded-lg", "transition-all", "duration-200", "hover:shadow", "outline-none", "focus:outline-none", "disabled:bg-none", "disabled:bg-lightgray", "disabled:text-darkgray", "disabled:shadow-none" ]
+button =
+  [ "px-6"
+  , "py-4"
+  , "rounded-full"
+  , "font-black"
```

Comment:
I think this should be `font-bold`, no?

Reply:
just checked on zepln, you are right. Changing